test/test_functional_tensor.py

```diff
@@ -62,10 +69,17 @@ def test_hflip(self):
         hflipped_img_script = script_hflip(img_tensor)
         self.assertTrue(torch.equal(hflipped_img, hflipped_img_script))
 
-    def test_crop(self):
-        script_crop = torch.jit.script(F_t.crop)
+    def test_hflip_cpu(self):
+        self._test_hflip("cpu")
+
+    @unittest.skipIf(not torch.cuda.is_available(), reason="Skip if no CUDA device")
+    def test_hflip_cuda(self):
+        self._test_hflip("cuda")
 
-        img_tensor, pil_img = self._create_data(16, 18)
+    def _test_crop(self, device):
+        script_crop = torch.jit.script(F.crop)
```

Comment:
Changed `F_t` to `F` as discussed https://github.com/pytorch/vision/pull/2568/files#r468534180